src/Pages/Signup/Signup.css

```diff
@@ -0,0 +1,14 @@
+.signupContainer{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
```

Comment:
You can avoid writing these three lines by just writing "place-items: center"!

Reply:
Thanks Neha. But It didn't work with just one line. I still have to write "justify-content: center" to get the desired output